test/Backtracing/FatalError.swift

```diff
@@ -8,7 +8,7 @@
 // UNSUPPORTED: asan
 // REQUIRES: executable_test
 // REQUIRES: backtracing
-// REQUIRES: OS=macosx || OS=linux-gnu
+// REQUIRES: OS=linux-gnu
```

Comment:
In case we forgot to add it back, should we add some comments there explaining the context?
eg.
```suggestion
// FIXME: Temporarily disable this on macOS - some CI nodes need a software update 
// REQUIRES: OS=linux-gnu
```

Reply:
🙂 Don't worry, we aren't going to forget — I'm tracking this internally and should be told when the CI nodes get fixed.